### DISCUSSION TITLE
Rather Unfortunate Name for this software and website

### DISCUSSION THREAD

**Discussion by BYTEMEE1** [undergroundwires/privacy.sexy#106] [General]
Privacy maybe "sexy" but is English your native language?

**Comment by ItsIgnacioPortal** [comment]
I'm pretty sure the Dev knew exactly what they were doing when they named it privacy.sexy

**Comment by undergroundwires** [comment]
Right, English is not my native language. Sexy in tech world is often a fancy way of saying "good" and nothing sexual, even though I think privacy is sexually sexy too :)

I would appreciate ideas for potential names to rename the project.

Problems with current name include:
- There has been issues with search engines for this name. DuckDuckGo rejects to index it. It already got banned from Microsoft Bing (I think it's more about the content than the name though).
- Some corporate networks block .sexy TLD.
- It costs 40$ a month which pretty high for a hobby project without any capitalization.
  
You're right @PinkDev1 . I simply search for available names with `privacy` and found the name. Then the domain name gave the project its name.

**Comment by BYTEMEE1** [comment]
How about "Nuke Telemetry" or "Privacy Power" or "Stealth PC." or "Cloak". Lots of  possbilities out there....

**Comment by iLiketurtles3000** [comment]
privatefuture.xyz
privacy.xyz
privacyproof.xyz

few ideas :)

**Reply by undergroundwires** [reply]
privacy.xyz was taken, would be great though. Maybe the owner stole the idea from you? 😛 The name privacyproof is great and my favorite so far. Thank you for the contribution.

**Reply by ItsIgnacioPortal** [reply]
[xyz TLDs are supper sketchy](https://www.bleepingcomputer.com/news/security/these-are-the-top-level-domains-threat-actors-like-the-most/). A more common TLD would be better than .xyz. But I personally like privacy.sexy as is

**Reply by TheAndr0id** [reply]
What about switching out the TLD.  privacy.sexy.org or privacy.sexy.net...  I don't know if they are cheaper to register, but my personal .ca domain runs me $12 (CAN) a year....

(I personally like the privacy.sexy app name, but I agree the .sexy TLD is gonna get people in corporate firewall logs....)

**Comment by undergroundwires** [comment]
Created privacylearn.com as alternative, see #470

**Comment by thirul95** [comment]
![Uploading IMG-20250225-WA0197.jpg…]()
